src/targets/stm32f1-generic/main.c

```diff
@@ -6,8 +6,28 @@
 #include <stm32f1xx.h>
 
 #include "platform/stm32f1/flash.h"
+#include "platform/stm32f1/gpio.h"
 #include "platform/stm32f1/rcc.h"
 #include "platform/stm32f1/systick.h"
+#include "platform/stm32f1/usb.h"
+
+#include "util/data.h"
+#include "util/types.h"
+
+#define CFG_TUSB_CONFIG_FILE "targets/stm32f1-generic/tusb_config.h"
```

Comment:
Do we need this in master?

Reply:
yes, when compiling platform and target sources the location of tusb_config.h is not in the include path, ofc I'd prefer if we didnt have to do this, it's just the workaround i found